handlers/graph.go

```diff
@@ -120,6 +123,14 @@ func isOutside(n *graph.Node, namespaces []string) bool {
 	return true
 }
 
+func isInaccessible(n *graph.Node, accessibleNamespaces map[string]bool) bool {
+	if _, found := accessibleNamespaces[n.Namespace]; !found {
+		return true
+	} else {
```

Comment:
This is an unnecessary else clause.  Not a blocker but would probably be better to just:
```
_, found := accessibleNamespaces[n.Namespace]
return !found
```